spec/tags/default_factory_spec.rb

```diff
@@ -164,5 +164,10 @@ def parse_options(text)
       expect(t.pair).to be_instance_of(Tags::DefaultTag)
       expect(t.pair.name).to eq "key"
     end
+
+    it "allows name only" do
```

Comment:
Realistically this test should be more descriptive-- technically these aren't "allowed".